projects/kit/components/radio/radio.template.html

```diff
@@ -10,7 +10,6 @@
     <div
         *ngIf="checked"
```

Comment:
You need to replace `*ngIf` with `[class.t-mark_visible]="checked"` and do transition, otherwise it immediately disappears.